client/my-sites/themes/theme-showcase.jsx

```diff
@@ -376,7 +379,7 @@ class ThemeShowcase extends Component {
 						uri={ this.constructUrl() }
 					/>
 				) }
-				<div className="themes__content">
+				<div className="themes__content" ref={ this.scrollRef }>
```

Comment:
So weird that this ref wasn't here before. What were we scrolling before?

Reply:
I think the scrollRef was missed during a code refactor.